pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -674,6 +692,8 @@ MetricsConfiguration metricsConfiguration() {
     metricsConfiguration.setMode(metricsMode);
     metricsConfiguration.setHost(metricsHost.toString());
     metricsConfiguration.setPort(metricsPort);
+    metricsConfiguration.setPushHost(metricsPushHost.toString());
+    metricsConfiguration.setPushPort(metricsPushPort);
```

Comment:
Is there a way to throw an error if the user configures a set of host / port values that don't correspond to the chosen mode?

Reply:
picoCLI doesn't let us see what CLI flags there are, just the resulting values and a default is indistinguishable from a flag with a default value.

Reply:
So, no.  Not with our current CLI/TOML library.

Reply:
Will be possible using what's in #679